src/app/search/search-filters.ts

```diff
@@ -1100,6 +1101,18 @@ export function searchFilters(
           })
         );
       },
+      hasMod(item: D2Item) {
+        return (
+          item.sockets &&
+          item.sockets.sockets.some((socket) => {
+            return !!(
+              (socket.plug || false) &&
+              ![2323986101, 2600899007].includes(socket.plug.plugItem.hash) &&
+              socket.plug.plugItem.itemTypeDisplayName.includes(' Mod')
```

Comment:
Is there a better thing to look for? :s

Reply:
will this work with i18n?

Reply:
itemType=19?
https://data.destinysets.com/i/InventoryItem:444600262

Reply:
ItemCategory:59

Reply:
Hmm, itemCategory part of `socket.plug.plugItem.itemCategoryHashes.includes(59)` seems to match all equippable items. The rest of the itemCategoryHashes don't seem to be any more useful. One is for the specific bucket, the other for weapon/armor sections. 

I've found that similar to masterwork, the `plugCategoryIdentifier` has a somewhat unique string id that I can use.